neuralprophet/utils.py

```diff
@@ -748,6 +748,7 @@ def configure_trainer(
     config: dict,
     metrics_logger,
     additional_logger: str = None,
+    accelerator: str = None,
```

Comment:
the idea is to disable GPU by default and only activate it if explicitly stated by the user. The background is: my feeling is that for most workloads the GPU will rather decrease training time instead of increasing it, due to the small batch sizes (in terms of bytes) forecasting workloads usually have. All the transfers to the accelerator and back will have negative effects for many workloads.